IdentityCore/src/broker_operation/request/account_request/MSIDBrokerOperationSignoutFromDeviceRequest.m

```diff
@@ -94,8 +96,9 @@ - (NSDictionary *)jsonDictionary
     }
     
     json[MSID_PROVIDER_TYPE_JSON_KEY] = MSIDProviderTypeToString(self.providerType);
-    json[MSID_SIGNOUT_FROM_BROWSER_KEY] = @(_signoutFromBrowser);
-    json[MSID_CLEAR_SSO_EXT_COOKIES_KEY] = @(_clearSSOExtensionCookies);
+    json[MSID_SIGNOUT_FROM_BROWSER_KEY] = [NSString stringWithFormat:@"%d", (int)_signoutFromBrowser];
```

Comment:
will this cause compatibility issue?

Reply:
ignore my comment, our code can support both NSString and NSNumber :)

Reply:
NSNumber actually doesn't even get sent when converted to query items :) So this was essentially broken and never sent in current version.